### PR TITLE
willys/branching-trunk-based

### PR DESCRIPTION
**Pull request by wpschaub**
Comtinue the discussion around trunk-based branching strategy, based on workshop script.

### CODE REVIEW

**Review by pmajor74 — DISMISSED** [review]
Looks good.

**Review by pmajor74 — DISMISSED** [review]

File: content/branching-trunk-based.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Trunk-based development is a proven and common practice among high-performing DevOps teams, often complemented by feature flags and/or ring-deployments to deploy continuously and release on demand – our future vision. 
+
+---
+
+# Getting started
+
+> ![Trunk](/images/branching-trunk-based-3.png)
+
+We start with a trunk, that has a hypothetical version of MAJOR 0, MINOR 1, and PATCH 0, aka **0.1.0**. ome point we create a feature branch to build a new feature and another feature branch to address a bug.
```
Comment (danielrbroderick) [inline content/branching-trunk-based.md]: Small typo on one point after 0.1.0

Comment (wpschaub) [reply content/branching-trunk-based.md]: Inserting the missing characters. Better?

**Review by littleswancode — APPROVED** [review]
Looks good @wpschaub

**Review by danielrbroderick — APPROVED** [review]

**Pull request merged**